### DISCUSSION TITLE
Moving from RoboFrom to KeepassXC - imported uid/pwd not found with browser integration

### DISCUSSION THREAD

**Discussion by mhdgraphics** [keepassxreboot/keepassxc#10164] [Q&A]
Hi! I recently exported my RoboForm data in .CSV format and imported it to KeepassXC 2.7.6.
I set up for browser integration but XC will say: "ERROR! No logins found! <https://www.example.com>" even though the entry is the same one I used on RoboForm without issue. It does not happen on every website login but I can't find where the differences are for the entries that don't work. I know I;m missing something.

**Comment by droidmonkey** [comment]
Make sure the url of the entry matches that of the website you are using. http vs https is also important.

**Reply by mhdgraphics** [reply]
Do they need to match exactly or by domain?
(Some sites have multiple servers)

**Reply by droidmonkey** [reply]
Match by topmost domain, so for example: 

`sub1.example.com` will not match `sub2.example.com`

`example.com` will match `sub1.example.com`
